dipy/reconst/dki.py

```diff
@@ -2336,6 +2336,7 @@ def mkt(self, *, min_kurtosis=-3.0 / 7, max_kurtosis=10):
             self.model_params, min_kurtosis=min_kurtosis, max_kurtosis=max_kurtosis
         )
 
+    @warning_for_keywords()
```

Comment:
This is not currently on master, but seems like it does need to be here. Or am I wrong?

Reply:
we do not need it here because it was already a keyword only arguments function.

The decorator is here to warn user from old functions.  New functions like this one do not need it.

Reply:
OK. Removed in da97f4b47